### ISSUE TITLE
Dynamo Customizer to Hydra

### ISSUE THREAD

**Issue by mostaphaRoudsari** [HydraShare/hydra#25]
@alfarok is possible to link [Dynamo Customizer](http://dynamobim.com/dynamo-customizer-beta-now-available/) to Hydra? We can add a new key to input.json where user can share the link and then we can embed the page in Hydra. That will be pretty cool IMO.

**Comment by tatlin** [comment]
Yes we discussed this after the hackathon. It would be very cool

-matt

**Comment by alfarok** [comment]
This is a great idea, I will start looking into it!  I also want to get a screenshot feature for the background 3D preview as soon as possible.  There is built in functionality for capturing the canvas but not geometry.  I have messed around with it a little but haven't quite figured out how to implement it into a custom UI node.  Even as a standalone feature it's something I think many people would be excited about.

**Comment by mostaphaRoudsari** [comment]
@alfarok I agree and I remember some other users asked for it on Dynamo's github. Looking forward to it.
